IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcWellKnownText/Documentation.md

```diff
@@ -0,0 +1 @@
+_IfcWellKnownText_ is a geometric representation item that is defined by a text string in _WellKnowText_ the format of which is specified in ISO 19162.
```

Comment:
The attribute `IfcWellKnownText::WellKnownText`'s documentation and this documentation are not on on par. While the former speaks also of ISO 13249, this one does not.

Reply:
I see this has been fixed while reviewing :P (94965d3)